src/Microsoft.DotNet.Wpf/src/System.Printing/CPP/src/GDIExporter/gdibitmap.cpp

```diff
@@ -188,7 +188,7 @@ void CGDIBitmap::SetupPalette(interior_ptr<BITMAPINFO> bmi, int bitCount)
     }
     else
     {
-        Debug::Assert(false, "Unsupported format");
+        Debug::Fail("Unsupported format");
```

Comment:
For these, did the compilation process just fail before it got here, and that's why we only saw 4 errors? Or is there something else going on?

Reply:
Yes. Stephen pulled out 4 error messages from the build, but there are 15 changes here. I'm asking if the other 11 are just from further down in the build tree and we didn't attempt to build them, so we didn't see 15 errors, or if something more complex is going on.

Reply:
I think this is a search/replace for all Debug.Assert(false. I'm concerned the sdk build ended after those first four, and that even this set of 15 won't fix it, as I see lots of two-arg asserts with non "false" codinitions throughout the C++/CLI code.  But I also don't know what the overload resolution problem is yet and so don't know how to reason about it.